crowdin/model/projects.go

```diff
@@ -242,6 +259,98 @@ type ProjectsAddRequest struct {
+	// Enum 0, 1, 2, 3, 4, 5. Default: 0.
+	//  0 - Show – translators will translate each instance separately,
+	//  1 - Hide (regular detection) – all duplicates will share the same translation
+	//  2 - Show, but auto-translate them,
+	//  3 - Show within a version branch (regular detection) - duplicates will be hidden only
+	//      between versions branches
+	//  4 - Hide (strict detection) – all duplicates will share the same translation
+	//  5 - Show within a version branch (strict detection) - duplicates will be hidden only
+	//      between versions branches
+	TranslateDuplicates *int `json:"translateDuplicates,omitempty"`
+	// Delay workflow start after project creation. Default: false.
+	DelayedWorkflowStart *bool `json:"delayedTranslations,omitempty"`
+	// Defines whether to export only approved strings.
+	// Note: value greater than 0 can't be used with `exportStringsThatPassedWorkflow=true`
+	//       in same request.
+	ExportWithMinApprovalsCount *int `json:"exportWithMinApprovalsCount,omitempty"`
+	// Defines whether to export only strings that passed workflow.
+	// Note: `true` value can't be used with `exportWithMinApprovalsCount>0` in same request
+	// or in projects without an assigned workflow.
+	ExportStringsThatPassedWorkflow *bool `json:"exportStringsThatPassedWorkflow,omitempty"`
+	// Clear QA checks for translations with specific number of approvals. Default: 1.
+	QAApprovalsCount *int `json:"qaApprovalsCount,omitempty"`
+	// Custom QA checks identifiers.
+	CustomQACheckIDs []int `json:"customQACheckIds,omitempty"`
+	// MT Engine Identifier.
+	MTID int `json:"mtId,omitempty"`
+	// Fields.
+	Fields map[string]any `json:"fields,omitempty"`
+	// Target Languages Identifiers.
+	Languages []string `json:"languages,omitempty"`
+}
+
+// LanguageMapping represents a project language mapping.
+type LanguageMapping struct {
+	Name                 string `json:"name"`
+	TwoLettersCode       string `json:"two_letters_code"`
+	ThreeLettersCode     string `json:"three_letters_code"`
+	Locale               string `json:"locale"`
+	LocaleWithUnderscore string `json:"locale_with_underscore"`
+	AndroidCode          string `json:"android_code"`
+	OSXCode              string `json:"osx_code"`
+	OSXLocale            string `json:"osx_locale"`
+}
+
+// NotificationSettings represents a project notification settings.
+type NotificationSettings struct {
+	// Notify translators about new strings. Default: false.
+	TranslatorNewStrings *bool `json:"translatorNewStrings,omitempty"`
+	// Notify project managers about new strings. Default: false.
+	ManagerNewStrings *bool `json:"managerNewStrings,omitempty"`
+	// Notify project managers about language translation/validation completion.
+	// Default: false.
+	ManagerLanguageCompleted *bool `json:"managerLanguageCompleted,omitempty"`
+}
+
+// WorkflowTemplateStepConfig represents a workflow template
+// step configuration and is used in the ProjectsAddRequest.
+type WorkflowTemplateStepConfig struct {
```

Comment:
Please add a todo to move this config to the workflows service later https://developer.crowdin.com/enterprise/api/v2/#tag/Workflows